src/features/Students/data/slice.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { RequestStatus } from 'features/constants';
+
+const initialState = {
+  table: {
+    currentPage: 1,
+    data: [],
+    status: RequestStatus.LOADING,
+    error: null,
+    numPages: 0,
+    count: 0,
+  },
+  courses: {
```

Comment:
courses, classes and filters have the same initialState, is there some way to unify it? @AuraAlba

Reply:
I created a constat and import it in the states with the same structure